src/modules/control/launch/control.launch

```diff
@@ -9,9 +9,27 @@
     <param name="debug" value="true"/>
   </node>
 
-  <node type="can_control_adapter.py" pkg="control" name="can_controller" output="screen">
+  <node type="waypoint_rviz_visualizer.py" pkg="control" name="virtual_donut_node" output="screen">
+    <!-- callback time interval -->
+    <param name="callback_interval" value="0.025" />
+    <!-- point density desired for the donut waypoint visualizer -->
+    <param name="point_density" value="300" />
+    <param name="msg_scale" value="0.05" />
+    
+    <param name="inner_donut_r" value="1" />
```

Comment:
Can we make the default colors for inner and outer donuts different? Just in case we accidentally make the outer smaller than the inner sometime in the future.

Reply:
The inner one is pink now